### ISSUE TITLE
Is there a database we can use with this?

### ISSUE THREAD

**Issue by GiverofMemory** [erfg12/EQTrilogy-Old#28]
Glad to find this exciting project!  Is there a database we can use with this as sort of a starter?  I am assuming that PEQ is not really compatible at this point.  We have a database open source here: https://github.com/GiverofMemory/NostalgiaEQ-Database and it would be really cool if you did something like this so we can all collaborate and get classic trilogy in action!

**Comment by erfg12** [comment]
This source is based off EQEmu 4.4. The code itself really needs the most work. But the DB is I believe from the TAK Project, so around POK era. The DB would need the NPCs and items refined to the Velious era.

In the readme file there's a link http://erfg12.github.io/EQClassic/ which has a direct link to a database dump that I did earlier today. But we could do a code repo for it I suppose.

Because this code is based off 4.4 I would really like to get this project working: https://github.com/erfg12/Trilogy It's EQMacEmu's code which I forked and tried adding all the op codes for EQClassic, but it still has issues. It's much closer to current day EQEmu and therefore doesn't have all the bugs or NPC move quirks that Yeahlight started developing.
